### PR TITLE
Fixes some grammar problems in Js Hoisting

### PR DESCRIPTION
**Pull request by mmdbalkhi**
#### Summary

Fixes some grammar problems in Js Hoisting

#### Motivation

Fix some grammar problems (There are really minor problems!)

#### Supporting details

N/A

#### Related issues

N/A

#### Metadata

- [ ] Adds a new document
- [ ] Rewrites (or significantly expands) a document
- [X] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/glossary/hoisting/index.md [hunk 1]
```diff
@@ -119,8 +119,8 @@ However the class is not initialized by default, so any code that uses it before
 
 [Function expressions](/en-US/docs/Web/JavaScript/Reference/Operators/function) and [class expressions](/en-US/docs/Web/JavaScript/Reference/Classes#class_expressions) are not hoisted.
 
-The expressions evaluate to a function or class (respectively), which are typically assigned to a variable.
-In this case the variable declaration is hoisted and the expression is its initialization.
+The expressions evaluate a function or class (respectively), which is typically assigned to a variable.
```
Comment (Josh-Cena) [inline files/en-us/glossary/hoisting/index.md:122]: The original text is correct. Expressions "evaluate to" values.

**Review by Josh-Cena — CHANGES_REQUESTED** [review]

File: files/en-us/glossary/hoisting/index.md (hunk 1 above)
Comment (mmdbalkhi) [reply files/en-us/glossary/hoisting/index.md]: @Josh-Cena done

**Review by Josh-Cena — APPROVED** [review]
This page will undergo some more significant rewrite in the near future, but it doesn't hurt to do these corrections now. Thank you, and congratulations on your first contribution here—welcome aboard!

File: files/en-us/glossary/hoisting/index.md [hunk 2]
```diff
@@ -119,8 +119,8 @@ However the class is not initialized by default, so any code that uses it before
 
 [Function expressions](/en-US/docs/Web/JavaScript/Reference/Operators/function) and [class expressions](/en-US/docs/Web/JavaScript/Reference/Classes#class_expressions) are not hoisted.
 
-The expressions evaluate to a function or class (respectively), which are typically assigned to a variable.
-In this case the variable declaration is hoisted and the expression is its initialization.
+The expressions evaluate to a function or class (respectively), which is typically assigned to a variable.
```
Comment (Josh-Cena) [inline files/en-us/glossary/hoisting/index.md:122]:
```suggestion
The expressions evaluate to a function or class (respectively). They are typically then assigned to a variable or passed to other functions.
```

**Pull request merged**